src/sentry/utils/math.py

```diff
@@ -30,3 +30,33 @@ def mad(values, K=1.4826):
     # http://en.wikipedia.org/wiki/Median_absolute_deviation
     med = median(values)
     return K * median([abs(val - med) for val in values])
+
+
+def nice_int(x):
+    """
+    Round up to the nearest "nice" number.
+    """
+
+    if x == 0:
+        return 0
+
+    exp = int(math.log10(x))
+
+    if x < 10:
+        rounded = 10 ** exp
```

Comment:
Won't `rounded` always be 1 as `int(math.log10())` on values < 10 is always 0, giving `10 ** 0`?

Reply:
That's a good point. I can add this in as a minor optimization here as well as the case where x < 100.